perfgraph.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Should rename this file `perfgraph.sh` --> `profiler` (note no `.sh`), and explain it in readme